proxy/src/main/resources/default-velocity.toml

```diff
@@ -33,8 +33,8 @@ prevent-client-proxy-connections = false
 #                  Velocity's native forwarding. Only applicable for Minecraft 1.13 or higher.
 player-info-forwarding-mode = "NONE"
 
-# If you are using modern or BungeeGuard IP forwarding, configure a unique secret here.
-forwarding-secret = ""
+# If you are using modern or BungeeGuard IP forwarding, configure a file that contains a unique secret here.
+forwarding-secret-file = ""
```

Comment:
Nit: could we default this to `forwarding.secret` as in `VelocityConfiguration`? This will save the extra re-save during the initial start-up.